src/Ocis.php

```diff
@@ -115,13 +121,22 @@ public function __construct(
         $this->graphApiConfig = Configuration::getDefaultConfiguration()->setHost(
             $this->serviceUrl . '/graph'
         );
+        if($educationAcessToken !== '') {
+            $this->educationGuzzle = new Client(self::createGuzzleConfig($connectionConfig, $this->educationAcessToken));
+        }
     }
 
     public function getServiceUrl(): string
     {
         return $this->serviceUrl;
     }
 
+    public function setEducationAcessToken(string $acessTocken): void
```

Comment:
```suggestion
    public function setEducationAccessToken(string $acsessToken): void
```

Reply:
`acsessToken` has a typo in the above suggestion.
I will check what happened...